subprojects/docs/src/docs/userguide/organizingGradleProjects.adoc

```diff
@@ -145,19 +146,26 @@ It also provides direct access to the Gradle API. Additional dependencies can be
 ++++
 
 A typical project including `buildSrc` has the following layout.
+Any code under `buildSrc` should use a package similar to application code.
```

Comment:
This line is confusing. Does it mean the code under buildSrc should have a similar package name as your application code?

I know it doesn't but that's what it reads like. I'd remove this sentence. buildSrc is a java project like any other, so all the same best practices apply.